tests/conversions/fixtures.py

```diff
@@ -1618,6 +1632,7 @@
     "SearchMatrixPairs": [search_matrix_pairs],
     "SearchMatrixOffsets": [search_matrix_offsets],
     "StrictModeConfig": [strict_mode_config, strict_mode_config_empty],
+    "DecayParamsExpression": [decay_params_expression_optionals],
```

Comment:
it's more like a nit, but I would rather not introduce an additional key in this mapping and just replace one of the decay_params_expression in expression = grpc.Expression(...) with this one (or add it there)